src/144RGBcode/gol.ino

```diff
@@ -0,0 +1,19 @@
+boolean cells[6][12];
+
+int getNumberOfActiveNeighbors(int i, int j) {
+  int neighbors[8][2] = {
+        {i - 1, j - 1}, {i - 1, j}, {i - 1, j + 1},
+        {i, j - 1}, {i, j + 1},
+        {i + 1, j - 1}, {i + 1, j}, {i + 1, j + 1}
+    };
+    int numActiveCells = 0;
+    for (int m = 0; m < 8; m++) {
+      //int k = (neighbors[m][0] + size) % size;
+      //int l = (neighbors[m][1] + size) % size;
+
+      //if (cells[k][l]) {
+        //numActiveCells += 1;
+      //}
+    }
+    return numActiveCells;
+}
```

Comment:
Can you tell me for what purpose you added this function?

Reply:
That's a work in progress as I would love to have a [game of life](https://en.wikipedia.org/wiki/Conway%27s_Game_of_Life) on the display. It just looks fancy and is not too hard to implement. (had to do it again for EGdP anyways this year)

Reply:
Wow actually the idea is very nice. I think the GoL effect would look fancy.